internal/models/session_handler.go

```diff
@@ -0,0 +1,271 @@
+package models
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/SwissDataScienceCenter/renku-gateway/internal/gwerrors"
+	mapset "github.com/deckarep/golang-set/v2"
+	"github.com/labstack/echo/v4"
+	"github.com/redis/go-redis/v9"
+)
+
+
+type SessionHandlerOption func(*SessionHandler)
+
+type SessionHandler struct {
+	cookieTemplate           func() http.Cookie
+	sessionTTL               time.Duration
+	tokenStore               TokenStore
+	sessionStore             SessionStore
+	createSessionIfMissing   bool
+	recreateSessionIfExpired bool
+	contextKey               string
+	headerKey                string
+	idQueryKey                 string
```

Comment:
shouldn't gofmt indent this the same as the other lines?

Reply:
Yes if you remember to run it.